pkgs/swiftgen/README.md

```diff
@@ -0,0 +1,3 @@
+# Swiftgen
```

Comment:
Maybe add a unit test that does expect(42, myFunc(...)) and a coverage setup in the initial PR, then the contributor doesn't have to fiddle with that later.